devices/IWearRemapper/src/IWearRemapper.cpp

```diff
@@ -762,16 +762,33 @@ WearStatus IWearRemapper::getStatus() const
         return WearStatus::WaitingForFirstRead;
     }
 
+    // Logic for combining the status of all the sensors.
+    // The tricky part is deciding how to handle the mixed case of timeout and overflow.
+    // The WaigingForFirstRead is not considered since the data is supposed not to be streamed.
+    // TODO: For now, overflow is stronger.
+    WearStatus status = WearStatus::Ok;
```

Comment:
We should remember to update this type sooner or later to `WearableStatus` in the whole project.